codemods/jscodeshift/next/13/remove-get-static-props/index.ts

```diff
@@ -21,6 +21,18 @@ type LazyModFunction = [
 	Partial<Record<string, string>>,
 ];
 
+// @TODO
```

Comment:
why TODO?

Reply:
because its a built-in js method, but it was not supported by typescript. changing target in tsconfig did not help, so i've decided to not spend much time on it, and fix later.

Reply:
sure, let's revisit that later then.